proposals/lip-0061.md

```diff
@@ -517,14 +520,16 @@ An object following the schema `aggregateCommitSchema`, which can be added the n
 
 ```python
 def selectAggregateCommit() -> AggregateCommit:
-    # Note that the BFT parameters at height maxHeightCertified+1 are already authenticated by the previous certificate.
-    heightNextBFTParameters = getNextHeightBFTParameters(getBFTHeights().maxHeightCertified + 1)
-    if previous function call returns valid height:
-        nextHeight = min(heightNextBFTParameters-1, getBFTHeights().maxHeightPrecommitted)
-    elif previous function call returns Not Found error:
+    try:
+        # Note that the BFT parameters at height maxHeightCertified+1 are already authenticated by the previous certificate.
+        heightNextBFTParameters = max(getNextHeightBFTParameters(getBFTHeights().maxHeightCertified+1), MIN_CERTIFICATE_HEIGHT + 1)
+        nextHeight = min(heightNextBFTParameters -1, getBFTHeights().maxHeightPrecommitted)
```

Comment:
```suggestion
        nextHeight = min(heightNextBFTParameters, getBFTHeights().maxHeightPrecommitted)
```

Reply:
If I'm not mistaken it is possible to remove some extra floating +- 1s

Reply:
That is not correct as `getBFTHeights().maxHeightCertified + 1` is input to a function.

Reply:
ah missed it!

Reply:
can be resolved from my side